src/app/organization/sms-campaigns/view-sms-campaign/campaign-tab/campaign-tab.component.scss

```diff
@@ -0,0 +1,14 @@
+.container {
+  max-width: 37rem;
+
+    .content {
+      div {
+        margin: 1rem 0;
+        word-wrap: break-word;
+      }
+    }
+  }
+  
+.mat-raised-button {
```

Comment:
there's no need for this, try to use as less scss as possible.

Reply:
@OussEmaDevCode I had to use this because otherwise the button was just touching the navbar but I have removed it now..